rockspec/lua-resty-hmac-ffi-1.0-0.rockspec

```diff
@@ -0,0 +1,24 @@
+package = 'lua-resty-hmac-ffi'
+version = '1.0-0'
+
+source = {
+  url = 'git://github.com/jamesmarlowe/lua-resty-hmac',
+  tag = 'v1.0'
+}
+
+description = {
+  summary = 'Lua driver for making and receiving hmac signed requests',
```

Comment:
Could we try to match this to the README like `HMAC functions for ngx_lua and LuaJIT`?

Reply:
fixed